ci/buildspec.yml

```diff
@@ -28,8 +28,9 @@ phases:
       # integration test is broken: https://github.com/awslabs/mxnet-model-server/issues/437
       #- python -m pytest mms/tests/integration_tests
       - pylint -rn --rcfile=./mms/tests/pylintrc mms/.
+      - python setup.py bdist_wheel --universal
```

Comment:
And also in setup.py seems requires "importlib2", how our build work without that library?